### PR TITLE
Add tab key for auto complete

### PR DESCRIPTION
**Pull request by fanwgwg**
Allow user to press `tab` to auto complelte/correct what he has input

### CODE REVIEW

File: src/main/java/seedu/agendum/ui/CommandBox.java [hunk 1]
```diff
@@ -110,6 +113,25 @@ public void handle(KeyEvent event) {
             }
         });  
     }
+    
+    private void registerTabKeyEventFilter() {
+        commandTextField.addEventFilter(KeyEvent.KEY_PRESSED, new EventHandler<KeyEvent>() {
+            public void handle(KeyEvent event) {
+                KeyCode keyCode = event.getCode();
+                if (keyCode.equals(KeyCode.TAB)) {
+                    System.out.println("tab pressed");
```
Comment (burnflare) [inline src/main/java/seedu/agendum/ui/CommandBox.java]: remove this?

Comment (fanwgwg) [reply src/main/java/seedu/agendum/ui/CommandBox.java]: hahah yes forgot to do so

Comment (rachx) [reply src/main/java/seedu/agendum/ui/CommandBox.java]: There is a println command executed in CommandBox.java too XD.
But autocomplete is nicely done. Easier to use now

**Review by burnflare — APPROVED** [review]
Works great, with a bit of code cleanup this is good to go!

**Pull request merged**